packages/xod-client/src/debugger/containers/Debugger.jsx

```diff
@@ -79,11 +138,42 @@ class Debugger extends React.Component {
     );
   }
 
+  renderTabActions() {
+    const isDisabled = R.isEmpty(this.props.log);
+    const title = notDisabledTitle =>
```

Comment:
I don't think the title should be altered. The reason for disabling the buttons when the log is empty is obvious enough.